cmd/kubeadm/app/util/version.go

```diff
@@ -100,7 +103,9 @@ func KubernetesIsCIVersion(version string) bool {
 // Internal helper: returns normalized build version (with "v" prefix if needed)
 // If input doesn't match known version pattern, returns empty string.
 func normalizedBuildVersion(version string) string {
-	if kubeReleaseRegex.MatchString(version) {
+	if version == kubeadmVersion {
```

Comment:
seems like this logic should maybe be in `KubernetesReleaseVersion` instead?

Reply:
@pipejakob what do you think?